src/parser.ts

```diff
@@ -2,13 +2,15 @@ import { defaultErrorMap, ZodErrorMap } from "./defaultErrorMap";
 import { INVALID, util } from "./helpers/util";
 import { ZodDef, ZodNever, ZodPromise } from "./index";
 import { NOSET, PseudoPromise } from "./PseudoPromise";
-import * as z from "./types/base";
```

Comment:
I really like that you also remove a case like this, when everything is imported (it was asking for circular dependencies' issue)